pkg/peer/config/constants.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+const (
+	// GlobalMSPID is used as the consortium-wide MSP ID (i.e. non org-specific)
+	GlobalMSPID = "general"
```

Comment:
btw - what does Fabric usually use for this ID?

Reply:
Fabric doesn't have this concept. This is a key that we use for non-MSP-specific config.